tritran/sqs-queue-bundle/2.0/config/packages/sqs_queue.yaml

```diff
@@ -0,0 +1,3 @@
+tritran_sqs_queue:
+    sqs_queue:
+        queues: ~
```

Comment:
What would happen if this config was removed?

Reply:
Hi, It's just a sample structure of configuration. Basically, there are not any error which belongs to.
BTW, this PR is still WIP, sorry about that. I need to update something else.  Thanks

Reply:
Okey. So if I understand you correctly, you could remove this line and still get the same results?

If so, (Looking at the current state of the PR) I would suggest closing this PR because Flex is smart enough to install this bundle without a recipe. 

However, I would love if you added some good default config that will make sense for 90% of the users. If so, I'll be happy to merge this PR.

Reply:
Thanks for your comments. Seems I should close this PR.